frontend/src/Explorer/View/Common.purs

```diff
@@ -158,7 +160,10 @@ txBodyView (TxBodyViewProps props) =
             <<< map txFromView $ props ^. txbInputs
         , P.div
             [ P.className "to-hash-container bg-transaction-arrow" ]
-            <<< map txToView $ props ^. txbOutputs
+            [ P.div
+                [ P.className "to-hash-wrapper"]
+                <<< map txToView $ props ^. txbOutputs
```

Comment:
I don't think that the syntax is more complex, it is just another operator (`^.` instead of `.`).  We are using lenses all over the code and it would be good to be consistent furthermore. As long as we don't have any decision to use record accessor or lenses I would suggest to use one approach only (lenses in this case). 

With `CSE-75` we already have an issue to discuss the pros/cons of using Lenses and I'm open to change it if needed.